admin/database/postgres/postgres.go

```diff
@@ -1580,6 +1580,46 @@ func (c *connection) InsertAsset(ctx context.Context, organizationID, path, owne
 	return res, nil
 }
 
+func (c *connection) FindUnusedAssets(ctx context.Context, createdBefore time.Time, limit int) ([]*database.Asset, error) {
+	query := `SELECT a.* FROM assets a 
+	LEFT JOIN projects p 
+	ON a.id = p.archive_asset_id 
+	WHERE p.archive_asset_id IS NULL`
+	var args []any
+	if createdBefore.IsZero() {
+		query += " ORDER BY a.created_on DESC LIMIT $1"
+		args = []any{limit}
+	} else {
+		query += " AND a.created_on < $1 ORDER BY a.created_on DESC LIMIT $2"
+		args = []any{createdBefore, limit}
+	}
```

Comment:
1. Consider dropping the `if` condition since `createdBefore` is always passed and is not a pointer
2. Consider using `WHERE NOT EXISTS (SELECT ...` instead of the `LEFT JOIN`. It seems slightly weird to join on a field that you have `IS NULL` on.

Reply:
1. 
2. The `SELECT a.* FROM tblA  a JOIN tblB b ON a.id = b.id WHERE b.id IS NULL` is a very standard SQL statement to do `ANTI JOIN` IMO. However I see people seem to be preferring `WHERE NOT EXISTS` over `LEFT JOIN WHERE id IS NULL`. I can make the change.